Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -448,21 +448,24 @@ public class BrowserViewController: UIViewController, BrowserViewControllerDeleg
     Preferences.Wallet.defaultWallet.observe(from: self)
     
     // Lists need to be compiled before attempting tab restoration
-    
-    var contentBlockListTask: AnyCancellable?
-    contentBlockListTask = ContentBlockerHelper.compileBundledLists()
-      .receive(on: DispatchQueue.main)
-      .sink { [weak self] res in
-        if case .failure(let error) = res {
-          log.error("Content Blocker failed to compile bundled lists: \(error)")
-        }
-          
-        contentBlockListTask = nil
-        
-        self?.contentBlockListCompiled = true
-        self?.setupTabs()
-    } receiveValue: { _ in
-      log.debug("Content Blocker successfully compiled bundled lists")
+    Task(priority: .background) {
```

Comment:
If you want this to execute on a background queue then you have to use `Task.detached`.  Using the `background` priority doesn't execute things on the background, it simply tells the system that these tasks are very low priority and may not even need to be executed at all.

Reply:
Ahh good to know. Thanks for that...still getting used to async await. But in all honesty this stuff doesn't need to be on a detached thread all that much